src/commands/dev/dev.js

```diff
@@ -414,6 +423,8 @@ const validateGeoCountryCode = (arg) => {
   return arg.toUpperCase()
 }
 
+const IP_VERSION_6 = 6
```

Comment:
IMHO it's easier to read to use the 6 directly and disable the lint rule for magic numbers

Reply:
Agree.